src/shared/lib/appNameUtils.ts

```diff
@@ -60,5 +69,28 @@ export const getAppName = (
 		case 'if_':
 		case 'ios_feast_app':
 			return 'Feast';
+		case 'ed_':
+		case 'editions_pressreader':
+			return 'Editions';
+	}
+};
+
+/**
+ * @name getIsNativeAppFromBowser
+ * @description Use a Bowser instance to determine if the browser is a native os.
+ * @param browser - Bowser instance
+ * @returns {IsNativeApp} - 'android' or 'ios' if the browser matches Android or iOS, otherwise undefined
+ */
+export const getIsNativeAppFromBowser = (
+	browser: Bowser.Parser.Parser,
+): IsNativeApp | undefined => {
```

Comment:
I still think this is slightly confusing because it mentions Native when the code doesn't have anything to do with native app. Is Native os a term people use ? because the way I understand it it's  just the client apps that can be native or running on a browser

But I guess the return type is `IsNativeApp`.  Also it's pretty strange to me that a type with that name is not a boolean but that is not introduced in this PR.

In any case this is all about naming so I'll approve and leave it up to you if you are happy to merge it like this